src/configuration/dictionary.py

```diff
@@ -13,5 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+EVA_DIR = ""
```

Comment:
Why is this an empty string? Shouldn't this be loaded from the `configuration.yml` file?